### PR TITLE
c is strongly typed

### PR DESCRIPTION
**Pull request by rfair404**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -342,6 +342,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Why do C programmers write their letters in *bold*?
+
+**A:** Because they're *strongly* typed.
```
Comment (styfle) [inline readme.md]: I don't get it

Comment (rfair404) [reply readme.md]: C is a strongly typed language. Strong is the tag used to write bold.

Comment (styfle) [reply readme.md:347]: The single `*` is italic, maybe you meant to use `**` for bold?

File: readme.md [hunk 2]
```diff
@@ -342,6 +342,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Why do C programmers write their letters in *bold*?
```
Comment (styfle) [inline readme.md:345]: I wonder if this joke would be better to start with something like:

> Why do C programmers break keyboards so often?

Comment (leopinzon) [reply readme.md:345]: +1 
Jjajaja, made me laugh just to imagine them typing on those white tall old keyboards

Comment (Zo-Bro-23) [reply readme.md:345]: Love this, but yeah, it's too similar to the one we already have.

File: readme.md (hunk 1 above)
Comment (styfle) [inline readme.md:347]:
```suggestion
**A:** Because they're **strongly** typed.
```

File: readme.md (hunk 2 above)
Comment (styfle) [inline readme.md:345]:
```suggestion
**Q:** Why do C programmers write their letters in **bold**?
```

**Pull request merged**